app/controllers/admin/merchant_items_controller.rb

```diff
@@ -1,6 +1,16 @@
 class Admin::MerchantItemsController < Admin::BaseController 
   def index
-    @merchant = Merchant.find(params[:merchant_id])
+    @merchant = find_merchant
+  end
+
+  def new
+    @merchant = find_merchant
```

Comment:
this isn't ideal but i needed a merchant for the admin user to be able to create an item for that user